Content.Server/PDA/Ringer/RingerSystem.cs

```diff
@@ -0,0 +1,134 @@
+using Content.Server.Traitor.Uplink; //Refrences left in for future traitor PDA intergration
+using Content.Server.Traitor.Uplink.Components;
+using Content.Server.UserInterface;
+using Content.Shared.Interaction;
+using Content.Shared.PDA.Ringer;
+using Robust.Server.GameObjects;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Audio;
+using Robust.Server.Player;
+using Robust.Shared.Player;
+using System;
+using System.Collections.Generic;
+
+namespace Content.Server.PDA.Ringer
+{
+    public class RingerSystem : EntitySystem
+    {
+
+        private readonly Dictionary<string, string> _pathIndex = new Dictionary<string, string>();
+
+        public override void Initialize()
+        {
+            base.Initialize();
+
+            //General Event Subscriptions
+            SubscribeLocalEvent<RingerComponent, ComponentInit>(RandomizeRingtone);
+            //RingerBoundUserInterface Subscriptions
+            SubscribeLocalEvent<RingerComponent, RingerSetRingtoneMessage>(RingerUpdateRingtoneDriver);
+            SubscribeLocalEvent<RingerComponent, RingerPlayRingtoneMessage>(RingerPlayRingtone);
+            SubscribeLocalEvent<RingerComponent, RingerRequestUpdateInterfaceMessage>(UpdateRingerUserInterfaceDriver);
+
+            //Dictionary Path Addition
+            _pathIndex.Add("A","/Audio/Effects/RingtoneNotes/a.ogg");
+            _pathIndex.Add("A#","/Audio/Effects/RingtoneNotes/a#.ogg");
+            _pathIndex.Add("B","/Audio/Effects/RingtoneNotes/b.ogg");
+            _pathIndex.Add("C","/Audio/Effects/RingtoneNotes/c.ogg");
+            _pathIndex.Add("C#","/Audio/Effects/RingtoneNotes/c#.ogg");
+            _pathIndex.Add("D","/Audio/Effects/RingtoneNotes/d.ogg");
+            _pathIndex.Add("D#","/Audio/Effects/RingtoneNotes/d#.ogg");
+            _pathIndex.Add("E","/Audio/Effects/RingtoneNotes/e.ogg");
+            _pathIndex.Add("F","/Audio/Effects/RingtoneNotes/f.ogg");
+            _pathIndex.Add("F#","/Audio/Effects/RingtoneNotes/f#.ogg");
+            _pathIndex.Add("G","/Audio/Effects/RingtoneNotes/g.ogg");
+            _pathIndex.Add("G#","/Audio/Effects/RingtoneNotes/g#.ogg");
```

Comment:
Just make all 12 notes an enum (A -> G#) and then have an array where it can be indexed to get the ringtone.
This also means elsewhere you don't need to keep using the string representation and can just use the enum.